fe/java-udf/src/main/java/org/apache/doris/udf/JdbcExecutor.java

```diff
@@ -194,6 +194,29 @@ public void rollbackTrans() throws UdfRuntimeException {
         }
     }
 
+    public List<Object[]> getBlock(int batchSize, Object colsArray) throws UdfRuntimeException {
+        try {
+            ArrayList<Integer> colsTypes = (ArrayList<Integer>) colsArray;
+            Integer[] colArray = new Integer[colsTypes.size()];
+            colArray = colsTypes.toArray(colArray);
+            int columnCount = resultSetMetaData.getColumnCount();
+            curBlockRows = 0;
+            do {
+                for (int i = 0; i < columnCount; ++i) {
+                    if (colArray[i] > 0) {
```

Comment:
Add comment to explain the value saved in `colArray`. Like `> 0` means this is a hll type.

Reply:
Done